src/main/java/io/proximax/sdk/BlockchainApi.java

```diff
@@ -161,13 +167,38 @@ public synchronized NetworkType getNetworkType() {
    public URL getUrl() {
       return url;
    }
+
+   /**
+    * convenience method for signing of transactions
+    * 
+    * @param transaction the transaction to be signed
+    * @param signer signing account
+    * 
+    * @return signed transaction instance
+    */
+   public SignedTransaction sign(Transaction transaction, Account signer) {
```

Comment:
provided convenience method to sign transaction by an account without need to worry about network generation hash